src/test/kotlin/org/jetbrains/research/ml/ast/transformations/util/TransformationsTestWithSdk.kt

```diff
@@ -0,0 +1,83 @@
+package org.jetbrains.research.ml.ast.transformations.util
```

Comment:
I hope in the future we can avoid such copying of files, I fixed this in the next PR